test/method/testcda.py

```diff
@@ -11,21 +11,24 @@
 """Test the CDA method in cclib"""
 
 from __future__ import print_function
+
+import sys
 import os
 import logging
 import unittest
 
 import numpy
 
-from testall import getfile
+sys.path.append("..")
+from test_data import getdatafile

```

Comment:
> Don't newer pythons support relative imports with a dot notation?

Yes, but only within modules. If you change this to `from ..test_data import getdatafile` and try `python testcda.py`, this happens:

    SystemError: Parent module '' not loaded, cannot perform relative import